docs/extensibility/resources.md

```diff
@@ -45,6 +45,13 @@ The **general** section is required and contains basic information about the res
     - **disableCreate** - when set to `true`, it disables the **Create** button. Defaults to `false`.
     - **disableEdit** - when set to `true`, it disables the **Edit** button. Defaults to `false`.
     - **disableDelete** - when set to `true`, it disables the **Delete** button. Defaults to `false`.
+- **externalNodes** - a optional list of links to external websites.
+  - **category** - a category name
+  - **scope** - either `namespace` or `cluster`. Defaults to `cluster`.
+  - **icon** - an optional icon
```

Comment:
We need to provide an information where user can find a list of the available icons